docs/collect-import-export.rst

```diff
@@ -1,11 +1,25 @@
 Settings QR codes
-=================
+========================
 
-Collect's settings can be imported and exported using QR codes. These can be provided by servers (like ODK Central) or can be used to share settings from one device to another.
+Collect can read settings from or write settings to a QR code. Settings QR codes can be provided by servers (like ODK Central) or can be used to share settings from one device to another.
 
-.. tip ::
+.. _configuration-qr-code-management:
 
-  In ODK Central, you can right-click the App User QR code, save it as an image, print it, and place it in a training room. You can also send the QR code saved as an image via email or WhatsApp and ask data collectors to import it.
+Managing and distributing settings QR codes
+--------------------------------------------
+
+.. warning::
+
+  Settings QR codes generally include confidential information and should be kept private. Make sure to follow :doc:`Android security best practices <collect-security>`.
```

Comment:
I'd drop the android security sentence. It doesn't help people keep their QR codes secure.

Reply:
It's the most important thing because generally the QR code will be available from Collect. You don't want someone to pick up a dropped phone and be able to impersonate it.